artifacts/opensuse/leap/leap_test.go

```diff
@@ -28,9 +29,10 @@ var _ = Describe("OpenSUSE Leap", func() {
 				common.DefaultPreferenceEnv:   "opensuse.leap",
 			},
 			&api.ArtifactDetails{
-				SHA256Sum:         "0f7f09a9a083088b51aa365fe0e4310e6b156c2153d6aa03a77b81eee884e52a",
-				DownloadURL:       "https://download.opensuse.org/distribution/leap/15.6/appliances/openSUSE-Leap-15.6-Minimal-VM.x86_64-Cloud.qcow2",
-				ImageArchitecture: "amd64",
+				SHA256Sum:            "0f7f09a9a083088b51aa365fe0e4310e6b156c2153d6aa03a77b81eee884e52a",
+				DownloadURL:          "https://download.opensuse.org/distribution/leap/15.6/appliances/openSUSE-Leap-15.6-Minimal-VM.x86_64-Cloud.qcow2",
+				ImageArchitecture:    "amd64",
+				AdditionalUniqueTags: []string{"15-latest"},
```

Comment:
Won't this break when defining multiple artifacts of leap?

Reply:
If we defined multiple of the same major version but as suggested by the follow up commit removing 15.5 I'm not sure we want to do that?

Reply:
Otherwise I can pass the `15-latest` tag into `New` and only provide it to the actual latest minor version we are syncing?

Reply:
I didn't think of it earlier, but can you make it figure out the latest minor version by itself? Similar to how versions of Fedora are gathered? This way we could avoid looking for new minor versions manually.